entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -84,28 +89,108 @@ public static Filter getTenantIdEqFilter(String tenantId) {
   }
 
   private static Filter transform(AttributeFilter filter) {
+    try {
+      if (filter.hasAttributeValue()) {
+        if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getName()) && filter.getOperator() == Operator.EQ) {
```

Comment:
Can this be generalized for every multivalued/list attribute instead of special casing for labels?

Reply:
No since we don't know the type of the field values in entity-service. We need to type the attributes used in entity-service and that involves having the entity-service connect to and make use of attribute service. For now we can get around this by treating attribute.labels specially.